src/Component/RuleGenerator/RuleGenerator.tsx

```diff
@@ -171,146 +172,157 @@ export const RuleGenerator: React.FC<RuleGeneratorProps> = (props) => {
+      />
+      <Form.Item
+        {...itemConfig}
+        label={locale.levelOfMeasurement}
+      >
+        <Radio.Group
+          onChange={onLevelOfMeasurementChange}
+          value={levelOfMeasurement}
+          buttonStyle="solid"
         >
-          <Radio.Group
-            onChange={onLevelOfMeasurementChange}
-            value={levelOfMeasurement}
-            buttonStyle="solid"
+          <Radio.Button
+            value="nominal"
           >
-            <Radio.Button
-              value="nominal"
-            >
-              {locale.nominal}
-            </Radio.Button>
-            <Radio.Button
-              value="cardinal"
-              disabled={attributeType === 'string'}
-            >
-              {locale.cardinal}
-            </Radio.Button>
-          </Radio.Group>
-        </Form.Item>
-        {
-          levelOfMeasurement !== 'cardinal' ? null :
-            <Form.Item
-              label={locale.classification}
-            >
-              <ClassificationCombo
-                classification={classificationMethod}
-                onChange={setClassificationMethod}
-              />
-            </Form.Item>
-        }
-        <Form.Item
-          label={locale.numberOfRules}
-          validateStatus={
-            classificationMethod === 'kmeans'
-              ? 'warning'
-              : numberOfRules < minNrClasses
-                ? 'error'
-                : undefined
-          }
-          help={classificationMethod === 'kmeans'
-            ? locale.numberOfRulesViaKmeans
+            {locale.nominal}
+          </Radio.Button>
+          <Radio.Button
+            value="cardinal"
+            disabled={attributeType === 'string'}
+          >
+            {locale.cardinal}
+          </Radio.Button>
+        </Radio.Group>
+      </Form.Item>
+      {
+        levelOfMeasurement !== 'cardinal' ? null :
+          <Form.Item
+            {...itemConfig}
+            label={locale.classification}
+          >
+            <ClassificationCombo
+              classification={classificationMethod}
+              onChange={setClassificationMethod}
+            />
+          </Form.Item>
+      }
+      <Form.Item
+        {...itemConfig}
+        label={locale.numberOfRules}
+        validateStatus={
+          classificationMethod === 'kmeans'
+            ? 'warning'
             : numberOfRules < minNrClasses
-              // eslint-disable-next-line max-len
-              ? `${locale.colorRampMinClassesWarningPre} ${minNrClasses} ${locale.colorRampMinClassesWarningPost}`
+              ? 'error'
               : undefined
+        }
+        help={classificationMethod === 'kmeans'
+          ? locale.numberOfRulesViaKmeans
+          : numberOfRules < minNrClasses
+            // eslint-disable-next-line max-len
+            ? `${locale.colorRampMinClassesWarningPre} ${minNrClasses} ${locale.colorRampMinClassesWarningPost}`
+            : undefined
+        }
```

Comment:
I know this is not part of thos PR but would be nice if we would have this human readable.